includes/class-wcpdf-assets.php

```diff
@@ -203,16 +203,21 @@ public function backend_scripts_styles ( $hook ) {
 
 		if (
 			$hook === 'woocommerce_page_wc-admin' &&
-			class_exists( 'Automattic\WooCommerce\Admin\PageController' ) &&
-			\Automattic\WooCommerce\Admin\PageController::is_admin_page()
+			WPO_WCPDF()->order_util->is_wc_admin_page()
 		) {
 			wp_enqueue_script(
 				'wpo-wcpdf-analytics-order',
-				WPO_WCPDF()->plugin_url() . '/assets/js/analytics-order.build.js',
-				array( 'wp-hooks' ),
+				WPO_WCPDF()->plugin_url() . '/assets/js/analytics-order' . $suffix . '.js',
+				array( 'wp-hooks', 'wp-i18n' ),
 				WPO_WCPDF_VERSION,
 				true
 			);
+
+			wp_set_script_translations(
```

Comment:
Wouldn't be easy to pass the strings using `wp_localize_script`?

Eg.

https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/e4d69034f97da4ab9bbd2aa9fa058a4fa9bb4586/includes/class-wcpdf-assets.php#L224

https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/e4d69034f97da4ab9bbd2aa9fa058a4fa9bb4586/assets/js/debug-script.js#L53

Reply:
Indeed that is simpler! I'm not sure why I didn't think of that before. Thank you!